histogram/histogram_test.go

```diff
@@ -1,11 +1,13 @@
 package histogram
 
 import (
+	. "github.com/smartystreets/goconvey/convey"
 	"os"
+	"testing"
 	"time"
 )
 
-func ExampleHist() {
+func TestExampleHist(t *testing.T) {
```

Comment:
What's the reason to change this from an Example to a test? AFAIK tests don't have their `stdout` content asserted against the `// Output` stanza

Reply:
format test file although not used